builds/misc/images-release.yaml

```diff
@@ -355,11 +349,6 @@ jobs:
       - powershell: "scripts/windows/build/Publish-Branch.ps1 -Configuration:\"$(configuration)\" -PublishTests:$False -UpdateVersion"
         displayName: "Build ($(configuration))"
         name: build
-      - task: PowerShell@2
-        displayName: "Build iotedge-diagnostics"
-        inputs:
-          filePath: ./edgelet/build/windows/build-diagnostics.ps1
```

Comment:
if we aren't going to use this script anymore I think we can delete it

Reply:
yep. I deleted the linux one, missed windows